modules/identifying-the-differences-in-allowed-connections-between-project-versions.adoc

```diff
@@ -1,105 +1,23 @@
-
-.Procedure
-
-* Run the following command to determine the connectivity differences between the Kubernetes manifests in the specified directories:
-+
-[source,terminal,subs="+quotes"]
-----
-$ roxctl netpol connectivity diff --dir1=_<folder1>_ --dir2=_<folder2>_     <1>
-----
-+
-<1> Specify the actual path to the directories containing the Kubernetes manifests you want to compare.
-+
-[NOTE]
-====
-The command considers all YAML files that you can accept using `kubectl apply -f`, and then these become valid inputs for your `roxctl netpol connectivity diff` command.
-====
-+
-The semantic difference report gives you an overview of the connections that were changed, added, or removed in `dir2` compared to the connections allowed in `dir1`. When you review the output, each line represents one allowed connection that was added, removed, or changed in `dir2` compared to `dir1`.
-If applicable, the `workloads-diff-info` provides additional details about the workloads associated with the added or removed connections.
-
-[id="distinguishing-between-syntactic-and-semantic-difference-outputs_{context}"]
-== Distinguishing between syntactic and semantic difference outputs
-
-In the following example, `dir1` is `netpol-analysis-example-minimal/`, and `dir2` is `netpol-diff-example-minimal/`. The difference between the directories is a small change in the network policy `backend-netpol`.
-
-.Example policy from `dir1`:
-
-[source,yaml]
-----
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  creationTimestamp: null
-  name: backend-netpol
-spec:
-  ingress:
-  - from:
-    - podSelector:
-        matchLabels:
-          app: frontend
-    ports:
-    - port: 9090
-      protocol: TCP
-  podSelector:
-    matchLabels:
-      app: backendservice
-  policyTypes:
-  - Ingress
-  - Egress
-status: {}
-----
-
-The change in `dir2` is an added `-` before the ports attribute, which produces a difference output.
-
+:FeatureName: Build-time network policy connectivity difference
+include::snippets/technology-preview.adoc[]
 
-[id="syntactic-difference-output_{context}"]
-=== Syntactic difference output
+This command helps you understand the differences in allowed connections between two project versions. It analyses the workload and Kubernetes network policy manifests located in each version's directory and creates a representation of the differences in either text or graphical format.
 
-.Procedure
+You can view connectivity difference reports in a variety of output formats, including `text`, `md`, and `csv`.
 
-* Run the following command to compare the contents of the `netpols.yaml` files in the two specified directories:
-+
-----
-$ diff netpol-diff-example-minimal/netpols.yaml netpol-analysis-example-minimal/netpols.yaml
-----
-+
-.Example output
-+
-[source,terminal,subs="+quotes"]
-----
-12c12
-<   - ports:
----
->     ports:
-----
+The following are examples of the different output formats:
```

Comment:
This part with the examples (links) to various output formats, should be moved to the example (after the link o the output in `md`, add also the links to the additional output formats - text and csv)